### PR TITLE
Refactor unit tests

### PR DESCRIPTION
**Pull request by Ramen-Noodle**
Updated unit tests to use setup class instead of __init__, removed old testing files

### CODE REVIEW

File: test/test_endpoint_operations.py [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+        self.api.setToken(self.rodsadmin_bearer_token)
+        
+        self.assertTrue(True)
+
+  
+if __name__ == '__main__':
+   unittest.main()
+
+
+
```
Comment (korydraughn) [inline test/test_endpoint_operations.py]: Please remove these lines.

File: test/test_endpoint_operations.py [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+    #     'op': 'create_user',
+    #     'name': cls.rodsuser_username,
+    #     'zone': cls.zone_name
+    # })
+    # cls.logger.debug(r.content)
+    # if r.status_code != 200:
+    #     cls._class_init_error = True
+    #     cls.logger.debug(f'Failed to create rodsuser [{cls.rodsuser_username}].')
+    #     return
+    # cls._remove_rodsuser = True
```
Comment (korydraughn) [inline test/test_endpoint_operations.py]: If these lines aren't needed anymore, go ahead and remove them.

File: test/test_endpoint_operations.py [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+    # r = requests.post(f'{cls.url_base}/authenticate', auth=(cls.rodsuser_username, config.test_config['rodsuser']['password']))
+    # cls.logger.debug(r.content)
+    # if r.status_code != 200:
+    #     cls._class_init_error = True
+    #     cls.logger.debug(f'Failed to authenticate as rodsuser [{cls.rodsuser_username}].')
+    #     return
+    # cls.rodsuser_bearer_token = r.text
+
+
+    # cls.logger.debug('Class setup complete.')
```
Comment (korydraughn) [inline test/test_endpoint_operations.py]: If these lines aren't needed anymore, go ahead and remove them.

File: test/test_endpoint_operations.py [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+    #     'name': cls.rodsuser_username,
+    #     'zone': cls.zone_name
+    # })
+    # cls.logger.debug(r.content)
+    # if r.status_code != 200:
+    #     cls.logger.debug(f'Failed to remove rodsuser [{cls.rodsuser_username}].')
+    #     return
+
+
+
```
Comment (korydraughn) [inline test/test_endpoint_operations.py]: If these lines aren't needed anymore, go ahead and remove them.

**Review by korydraughn — COMMENTED** [review]
Looks good so far.

**Comment by korydraughn** [comment]
Squash to taste and attach issue numbers. No pounds.

**Comment by Ramen-Noodle** [comment]
Squashed

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by korydraughn** [comment]
Oh. Do the tests still pass?

**Comment by Ramen-Noodle** [comment]
> Oh. Do the tests still pass?

All still passing

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**